server/src/main/java/io/crate/metadata/Functions.java

```diff
@@ -190,17 +182,34 @@ private static FunctionImplementation resolveFunctionBySignature(FunctionName na
                                                                      List<DataType<?>> argumentTypes,
                                                                      List<Symbol> arguments,
                                                                      SearchPath searchPath,
-                                                                     Map<FunctionName, List<FunctionProvider>> candidatesByName) {
-        var candidates = getCandidates(name, searchPath, candidatesByName);
+                                                                     Map<FunctionName, List<FunctionProvider>> builtInFunctions,
+                                                                     Map<FunctionName, List<FunctionProvider>> udfs) {
+        var candidates = builtInFunctions.get(name);
```

Comment:
minor: why did you inline it instead of a static method?

Reply:
It was only used here and I initially thought I'd have to restructure more

Reply:
no worries, you can leave as is.